ts-client/tests/vault.test.ts

```diff
@@ -1,27 +1,36 @@
-import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import { Connection, Keypair, PublicKey, SYSVAR_CLOCK_PUBKEY, ParsedAccountData } from "@solana/web3.js";
 import SolanaTokenList from "@solana/spl-token-registry/dist/module/tokens/solana.tokenlist.json";
 import { Wallet, Provider } from "@project-serum/anchor";
 
 import Vault from "../src/vault";
 import { SOL_MINT } from "../src/constants";
+import { ParsedClockState } from "../types/clock_state";
 import { airDropSol } from './utils';
 
 const mockWallet = new Wallet(new Keypair());
 const mainnetConnection = new Connection("https://api.mainnet-beta.solana.com");
 const devnetConnection = new Connection("https://api.devnet.solana.com/");
 const SOL_TOKEN_INFO = SolanaTokenList.tokens.find(token => token.symbol === 'SOL');
+var currentTime;
```

Comment:
nothing wrong with `var`, but we prefer `let` for lexical scoping now
you dont have to change just fyi

Reply:
Fixed